randovania/data/json_data/prime2.txt

```diff
@@ -9563,8 +9567,13 @@ Asset id: 2722128775
   * Normal Door to Central Area Transport West/Door to Watch Station
   > Pickup (Dark Ammo)
       All of the following:
-          Single Room Out of Bounds (Expert) and Standable Terrain (Expert) and Screw Attack into Tunnels/Openings (Expert) and Use Screw Attack (Space Jump)
-          Boost Ball or Morph Ball Bomb
+          Disabled Room Randomizer
```

Comment:
No paths to the pickup in case of Room Randomizer

Reply:
If it's room rando, youd just get it with spider ball from the other nodes